grammar_test/expressions.txt

```diff
@@ -164,9 +164,21 @@ x, *args = [1, 2]
 ---
 
 (program
-  (assignment (assignment_list (identifier) (identifier)) (array (integer) (integer)))
-  (assignment (assignment_list (identifier) (rest_assignment)) (array (integer) (integer)))
-  (assignment (assignment_list (identifier) (rest_assignment (identifier))) (array (integer) (integer))))
+  (assignment (left_assignment_list (identifier) (identifier)) (array (integer) (integer)))
+  (assignment (left_assignment_list (identifier) (rest_assignment)) (array (integer) (integer)))
+  (assignment (left_assignment_list (identifier) (rest_assignment (identifier))) (array (integer) (integer))))
+
+==========
+multiple assignment with multiple right hand sides
+==========
+
+x, y = foo, bar
```

Comment:
It is. For `y` to get assigned to anything, `foo` would have to return an array with at least 2 elements.